Toxygates/src/panomicon/Authentication.scala

```diff
@@ -60,11 +64,11 @@ class Authentication {
   }
 
   def getJwtToken(cookies: Array[Cookie]): Either[(JWT, String), String] = {
-    try {
-      val tokenCookie = cookies.find(c => c.getName == "__Host-jwt").getOrElse(
-        return Right("JWT cookie __Host-jwt was not set")
```

Comment:
Returning inside a try-block is generally considered bad (compiles to a throwable and can have unpredictable results)